client/src/main/scala/org/http4s/client/middleware/GZip.scala

```diff
@@ -0,0 +1,29 @@
+package org
+package http4s
```

Comment:
Usually keep `org.http4s` on one line, or else you're importing everything else in `org.*`.